examples/AnnotatorForClinicalDataExample.java

```diff
@@ -0,0 +1,543 @@
+		Response<Map<String, AcdProfile>> resp = sc.execute();
+		Map<String, AcdProfile> profileMap = resp.getResult();
+
+		for (String id:profileMap.keySet())
+			System.out.println("Profile Id: " + id + " Desc:" + profileMap.get(id).description());
+
+	}
+
+	// GET /profiles/{id}
+	public static void getProfile() {
+
+		GetProfileOptions options = new GetProfileOptions.Builder()
+				.id("wh_acd.ibm_clinical_insights_v1.0_profile").build();
+		ServiceCall<AcdProfile> sc = acd.getProfile(options);
+		Response<AcdProfile> response = sc.execute();
+		AcdProfile profile = response.getResult();
+		System.out.println(profile);
+
+		System.out.println("Id: " + profile.id());
+		System.out.println("Name: " + profile.name());
+		System.out.println("Annotators:");
+		List<Annotator> annos = profile.annotators();
+		for (Annotator anno:annos)
+			System.out.println('\t' + anno.name());
+
+	}
+
+	// POST /profiles
+	public static void createProfile() {
+
+		Map<String, Set<String>> conceptDetectionParams
+			= new HashMap<String,Set<String>>(){{put("libraries",
+				new HashSet<String>() {{ add("umls.latest"); }} ); }};
+
+		Annotator conceptDetection = new Annotator.Builder()
+				.name(Annotator.Name.CONCEPT_DETECTION)
+				.parameters(new java.util.HashMap<String,List<String>>(){{put("libraries", Arrays.asList("umls.latest")); }})
+				.build();
+
+		Builder bldr = new CreateProfileOptions.Builder();
+		bldr.id("my_profile");
+		bldr.name("My profile");
+		bldr.description("My profile desc");
+		bldr.addAnnotators(conceptDetection);
+
+		ServiceCall<Void> call = acd.createProfile(bldr.build());
+		Response<Void> resp = call.execute();
+		System.out.println("Response code: " + resp.getStatusCode()); // should be 201
+
+	}
+
+	// PUT /profiles
+	public static void updateProfile() {
+
+		Map<String, Set<String>> conceptDetectionParams
+			= new HashMap<String,Set<String>>(){{put("libraries",
+			   new HashSet<String>() {{ add("umls.2019AA"); }} ); }};
+
+		Annotator conceptDetection = new Annotator.Builder()
+				.name(Annotator.Name.CONCEPT_DETECTION)
+				.parameters(conceptDetectionParams)
+				.build();
+
+		UpdateProfileOptions.Builder bldr
+			= new UpdateProfileOptions.Builder();
+
+		bldr.id("my_profile");
+		bldr.newId("my_profile");
+		bldr.newName("My profile - updated");
+		bldr.newDescription("My profile desc - updated");
+		bldr.newAnnotators(Arrays.asList(new Annotator[] {conceptDetection}));
+
+		ServiceCall<Void> call = acd.updateProfile(bldr.build());
+		Response<Void> resp = call.execute();
+		System.out.println("Response code: " + resp.getStatusCode()); // should be 200
+
+	}
+
+	// DELETE /profiles/{id}
+	public static void deleteProfile() {
+
+		DeleteProfileOptions options = new DeleteProfileOptions
+				.Builder()
+				.id("my_profile")
+				.build();
+
+		ServiceCall<Void> call = acd.deleteProfile(options);
+		Response<Void> resp = call.execute();
+		System.out.println("Response code: " + resp.getStatusCode());
```

Comment:
For consistency, should we include a comment with the expected status code like we did in the previous two methods?